From ce78ee785b518da4a38b9866f93e5279f0870f3e Mon Sep 17 00:00:00 2001
From: Alexi Demers <alexidemers@gmail.com>
Date: Thu, 19 Jan 2023 21:13:53 -0500
Subject: [PATCH] rk3328-orangepi-r1-plus-common.dtsi: fix support for LTS eth1

Fix reset GPIOs and pins for Orangepi R1+ LTS. Took the information from
the corresponding devicetree in kernel 5.10
---
 .../boot/dts/rockchip/rk3328-orangepi-r1-plus-common.dtsi      | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus-common.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus-common.dtsi
index cadd0cd5b0..c4f043683d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus-common.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus-common.dtsi
@@ -254,9 +254,6 @@
 	phy-handle = <&rtl8211f>;
 	phy-mode = "rgmii";
 	phy-supply = <&vcc_phy>;
-	snps,reset-active-low;
-	snps,reset-delays-us = <0 10000 30000>;
-	snps,reset-gpio = <&gpio1 RK_PC2 GPIO_ACTIVE_LOW>;
 	snps,aal;
 	snps,rxpbl = <0x4>;
 	snps,txpbl = <0x4>;
-- 
2.34.1

